distribution/docs/src/main/resources/content/_managing/_configuring/configuring-multi-factor-auth.adoc

```diff
@@ -0,0 +1,24 @@
+:title: Configuring Multi Factor Authentication
+:type: subConfiguration
+:status: published
+:parent: Configuring REST Services for Users
+:summary: Configuring Multi Factor Authentication using Keycloak as an IdP.
+:order: 23
+
+// This is for MFA
+== {title}
+
+Mutli-factor authentication, sometimes referred to as two-factor authentication, allows for greater security.
+It does this by requiring users to provide multiple proofs of identity, typically through something they know, and something they have/are.
```

Comment:
✏️ consider rewording this. It is kind of confusing/non-descriptive

Reply:
Do you have any suggestions? I could provide examples, I.E.
"It does this by requiring users to provide multiple proofs of identity, typically through something they know (such as a password they created), and something they have/are (such as a randomly generated pin number sent to one of their personal devices through a separate application)."
or something like that?